apps/modernization-ui/src/routes/AppRoutes.tsx

```diff
@@ -92,6 +93,8 @@ export const AppRoutes = () => {
 
                                 {config.features.pageBuilder.enabled && (
                                     <Route path="/page-builder" element={<PageBuilderContextProvider />}>
+                                        {/*  temp add this route for testing*/}
+                                        <Route path="page/:pageId?" element={<EditPage />} />
```

Comment:
Issue: the `EditPage` component will not be used. New features need to be added to the `Edit` component that is the new design aligned component.

Reply:
This was fixed as i was doing the review -- marking as resolved